### DISCUSSION TITLE
Fast Forward Feature

### DISCUSSION THREAD

**Discussion by sauradipdebnath** [skyline-emu/skyline#286] [Ideas]
Use fast Forward to skip most cutscenes at high speed

**Comment by MythicalPlayz** [comment]
So something like the fast forwarding button on PPSSPP ?

It does semm lile a good Idea tho I am not sure how would it work backend.

**Reply by Heliumrich** [reply]
Just an on-screen button to swap between two different speed limit I guess ? Like 100% and 200% 

There already is a disable frame throttling setting in skyline

**Reply by sauradipdebnath** [reply]
Keep on discussing on fast forward!

**Reply by sauradipdebnath** [reply]
Make the emulator work on 1gb/512mb ram

**Comment by sauradipdebnath** [comment]
And don't forget to add fast forward to then
